tox.ini

```diff
@@ -11,10 +11,23 @@ deps =
   lint: pylint~=2.3.1
   mypy: mypy~=0.711
 
+setenv =
+  PYTHONPATH={toxinidir}/opentelemetry-api/src/
```

Comment:
If you're doing this to get `opentelemetry` on the path for tests: would installing `opentelemetry-api` in the testenv work instead? [`usedevelop`](https://tox.readthedocs.io/en/latest/config.html#conf-usedevelop) may work for this.

Reply:
I believe it would. Why do we use skipsdist anyway?

Reply:
IIRC because we've got multiple packages under separate directories so tox didn't know what to install.